src/Message.php

```diff
@@ -13,6 +13,22 @@
 
 /**
  * Message is a data object that stores log message data.
+ *
+ * @psalm-type Backtrace = list<array{
+ *     file:string,
+ *     line:int,
+ *     function?:string,
+ *     class?:string,
+ *     type?:string,
+ *     args?:array,
+ *     object?:object,
```

Comment:
Keys "args" and "object" removed in `Logger::collectTrace()`.

Reply:
Psalm shows errors without these keys

```
ERROR: InvalidArrayOffset - src/Logger.php:316:31 - Cannot access value on variable $trace using offset value of 'object', expecting 'file', 'line', 'function', 'class' or 'type' (see https://psalm.dev/115)
                        unset($trace['object'], $trace['args']);
```

Can be solved by adding array to the type
```
* @psalm-type Backtrace = list<array{
 *     file:string,
 *     line:int,
 *     function?:string,
 *     class?:string,
 *     type?:string,
 * }&array>
```

Reply:
To method `Logger::collectTrace` pass parameter:

```
list<array{
	file:string,
	line:int,
	function?:string,
	class?:string,
	type?:string,
	args?:array,
	object?:object,
}>
```

..., but result of method is:

```
list<array{
	file:string,
	line:int,
	function?:string,
	class?:string,
	type?:string,
}>
```